### PR TITLE
Update plugin for vue3

### PR DESCRIPTION
**Pull request by dev387**
- Ref [T95559](https://phabricator.nsoft.ba/T95559)

### CODE REVIEW

**Review by dikaso — APPROVED** [review]

File: generator/template/default/_browserslistrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-> 1%
-last 2 versions
-not dead
```
Comment (Lanchi) [inline generator/template/default/_browserslistrc:3]: Please explain why are we removing this?

File: generator/index.js [hunk 2]
```diff
@@ -4,15 +4,14 @@ const rimraf = require('rimraf');
 module.exports = (api, options, rootOptions) => {
 
   const pathsToRemove = [
-    'src/router.js',
-    'src/store.js',
```
Comment (Lanchi) [inline generator/index.js:8]: Are these files no longer generated by Vue CLI?

Comment (dev387) [reply generator/index.js:8]: Nope, at least with default selection.
This is what it generates:
image

File: generator/template/default/_browserslistrc (hunk 1 above)
Comment (dev387) [reply generator/template/default/_browserslistrc:3]: This is already defined in package.json file generated by vue-cli.
image

Comment (Lanchi) [reply generator/template/default/_browserslistrc:3]: Hmm I think this is a matter of choice done during initalization phase - user can choose does he wish to keep these settings (along with others) in package.json or in dedicated files.

Comment (dev387) [reply generator/template/default/_browserslistrc:3]: Yes, user can choose where to keep settings, but if you use default vue-3 config there is no prompt which ask you where you want save those settings.

**Review by Lanchi — APPROVED** [review]

**Pull request merged**